app/user.go

```diff
@@ -0,0 +1,560 @@
+package app
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"github.com/temporalio/tcld/protogen/api/auth/v1"
+	"github.com/temporalio/tcld/protogen/api/authservice/v1"
+	"github.com/urfave/cli/v2"
+	"strings"
+)
+
+const (
+	userIDFlagName              = "user-id"
+	userEmailFlagName           = "user-email"
+	accountRoleFlagName         = "account-role"
+	namespacePermissionFlagName = "namespace-permission"
+	permissionFlagName          = "permission"
```

Comment:
Is this flag used anywhere? I tried looking and it seems unused.

Reply:
Removed.